src/components/ProductForm/ProductForm.tsx

```diff
@@ -0,0 +1,253 @@
+                      >
+                        C4
+                      </div>
+                      <div
+                        className="c5"
+                        onClick={() => setFieldValue("color", "c5")}
+                      >
+                        C5
+                      </div>
+                      <div
+                        className="c6"
+                        onClick={() => setFieldValue("color", "c6")}
+                      >
+                        C6
+                      </div>
+                      <div
+                        className="c7"
+                        onClick={() => setFieldValue("color", "c7")}
+                      >
+                        C7
+                      </div>
+                    </div>
+                  </Box>
+                  <Box margin={2}>
+                    <div>Image</div>
+                    <input type="file" />
+                  </Box>
+
+                  <Box margin={2}>
+                    <InputLabel id="categoryList">Category List</InputLabel>
+                    <Field
+                      component={TextField}
+                      type="text"
+                      name="category"
+                      select
+                      fullWidth
+                    >
+                      <MenuItem value={"c1"}>category 1</MenuItem>
+                      <MenuItem value={"c2"}>category 2</MenuItem>
+                      <MenuItem value={"c3"}>category 3</MenuItem>
+                    </Field>
+                  </Box>
+                  <Box margin={2}>
+                    <InputLabel id="productDes">Product Description</InputLabel>
+                    <Field
+                      component={TextField}
+                      type="text"
+                      name="productDes"
+                      multiline
+                      rows={5}
+                      fullWidth
+                    />
+                  </Box>
+                  <Box margin={2}>
+                    <InputLabel id="stockCount">Stock Count</InputLabel>
+                    <Field
+                      component={TextField}
+                      type="text"
+                      name="stockCount"
+                      fullWidth
+                    />
+                  </Box>
+                  <Box margin={2}>
+                    <Field
+                      component={DatePicker}
+                      name="date"
+                      label="Exp. Date"
+                      format="dd/MM/yyyy"
+                      fullWidth
+                    />
+                  </Box>
+                </Form>
+                {isSubmitting && <LinearProgress />}
+              </DialogContent>
+              <DialogActions>
+                <Button onClick={handleClose} color="primary">
+                  Cancel
+                </Button>
+                <Button onClick={submitForm} color="primary">
```

Comment:
give this a `type="submit"` instead of onClick and it'll invokes the original form submit handler internally